cmd/livepeer/starter/starter.go

```diff
@@ -1348,6 +1348,15 @@ func StartLivepeer(ctx context.Context, cfg LivepeerConfig) {
 					if *cfg.Network != "offchain" {
 						n.SetBasePriceForCap("default", core.Capability_SegmentAnything2, config.ModelID, autoPrice)
 					}
+				case "realtime-to-realtime":
```

Comment:
How about calling it `realtime-ai` or `realtime-inference` or `realtime-video-ai`?

Reply:
Naming is the hardest thing in computer science :)

For reference here are the names of the other pipelines, which generally follow <input-type>To<output-type>

https://github.com/livepeer/go-livepeer/blob/439057979b22c6e6c4d6d388c2a811ee13fbcda3/core/capabilities.go#L75-L81

> How about calling it realtime-ai or realtime-inference or realtime-video-ai?

To me, the `-ai` or `-inference` suffixes don't fit any of the existing patterns and don't convey information in this context since all other pipelines are AI inference. We may also have other types of 'live' pipelines, so I think we want to specify the output type here too.

Another thing I have been thinking - we have been using the `realtime` terminology but I think `live` fits a little better (and is more on-brand for Livepeer). Plus it is slightly shorter.

So one option might be to use a prefix, eg `live-` for `live-video-to-video`

This makes the naming easily extendable to other areas too, eg `live-audio-to-text` for live transcription, `live-segment-anything-2` etc

Reply:
@j0sh I like the live prefixed input output naming scheme.

Reply:
Renamed to `live-video-to-video` in https://github.com/livepeer/go-livepeer/pull/3210/commits/a92dacb663aa2f12a8446c457f8342877c3aa42e